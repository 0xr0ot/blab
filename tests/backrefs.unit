out = \(foo bar\) \1 \(baz\) \2 \(quux\) \3 " - " \1 \2 \3 
foo = "foo "
bar = "bar "
baz = "baz "
quux = "quux "
