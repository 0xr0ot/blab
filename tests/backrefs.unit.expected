foo bar foo bar baz baz quux quux  - foo bar baz quux 